src/renderer/components/+workloads-daemonsets/daemonset-details.tsx

```diff
@@ -58,6 +62,17 @@ export class DaemonSetDetails extends React.Component<Props> {
     this.metrics = null;
   });
 
+  @disposeOnUnmount
+  revisionLoader = autorun(async () => {
+    const { object: daemonSet } = this.props;
+    const revisionHistory = await controllerRevisionApi.getRevisions({
+      name: daemonSet.getName(),
+      namespace: daemonSet.getNs()
+    }) as ControllerRevision[];
```

Comment:
This cast should be unnecessary `getRevisions`'s return type should be `ControllerRevision[] | null`